### ISSUE TITLE
头数不为2次幂的问题

### ISSUE THREAD

**Issue by tianbuwei** [keezen/ntk_alibi#4]
大佬您好，请问下 再 head 不为2次幂的时候，为什么要 两部分进行拼接：slopes = torch.cat([slopes, torch.pow(extra_base, extra_powers)], dim=0)，这有什么依据吗，您能稍微解释下吗？
![image](https://github.com/keezen/ntk_alibi/assets/88369405/9fc8721f-8256-4ee9-9ae6-e549e62ad91c)
